pkg/config/config.go

```diff
@@ -52,6 +53,8 @@ type Configuration struct {
 	ExcludedCharts        []string `mapstructure:"excluded-charts"`
 	HelmExtraArgs         string   `mapstructure:"helm-extra-args"`
 	Debug                 bool     `mapstructure:"debug"`
+	Namespace             string   `mapstructure:"namespace"`
+	ReleaseLabel          string   `mapstructure:"release-label"`
```

Comment:
TODO: Add check for zero value for `ReleaseLabel` if `Namespace` is set.

Reply:
Also, this option can probably be removed in v3 when we can just look for resources that are owned by the helm release CRD:

https://github.com/helm/community/blob/master/helm-v3/003-state.md#the-release-object